examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -90,6 +92,12 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
         viewModel.retrieveCustomerInfo()
 
         Purchases.sharedInstance.getOfferingsWith(::showError, ::populateOfferings)
+
+        Purchases.sharedInstance.getAmazonLWAConsentStatus({
+            Log.i("PurchaseTester", "AmazonLWAConsentStatus Success: $it")
+        }, {
+            Log.e("PurchaseTester", "AmazonLWAConsentStatus Error: $it")
+        })
```

Comment:
Figured we dont want a UI for this since there is no need to need to check more than once per app launch

Reply:
Not sure if there's an easy way to only do this for Amazon configurations? I think in `OfferingFragment.onCreateView` we check if it's play store, so maybe we could do something similar over here

Reply:
Unless we want to see the `UNAVAILABLE` log when configured for the play store

Reply:
Changed. It's not the cleanest way but we're now checking we're using the SDK in Amazon mode before checking consent status.